_layouts/layout-page-sidenav-sharable.html

```diff
@@ -3,23 +3,22 @@
 ---
 
 <div class="row sidenav-container">
-  <div class="col is-3 is-hidden-touch sidenav">
+  <div class="col is-2 is-hidden-touch sidenav">
     <aside class="sgds-menu">
       <ul class="sgds-menu-list">
         {%- include sidenav.html -%}
       </ul>
     </aside>
   </div>
-  <div class="col is-8 is-12-touch">
+  <div class="col is-8 is-offset-1 is-12-touch">
```

Comment:
```suggestion
  <div class="col is-8 is-offset-1 is-12-mobile">
```
Suggests to use `is-12-touch` as when the view point < 1023px is-12-touch is making the width 100% and with the margin-left from offset-1 it is pushing the columns out of the viewport. 

`is-12-touch`
![image](https://user-images.githubusercontent.com/2618932/128290863-e821b064-daf6-4f8e-a394-26b5e61c2e0c.png)

`is-12-mobile`
![image](https://user-images.githubusercontent.com/2618932/128290930-c84facae-b7d1-496c-b310-97b63a7affd5.png)